src/coreclr/vm/debugdebugger.cpp

```diff
@@ -901,6 +852,60 @@ extern "C" void QCALLTYPE DebugDebugger_CustomNotification(QCall::ObjectHandleOn
     END_QCALL;
 #endif // DEBUGGING_SUPPORTED
 }
+
+extern "C" BOOL QCALLTYPE DebugDebugger_IsLoggingHelper()
```

Comment:
This can be a `SuppressGCTransition` QCall. The `IsLoggingEnabled()` is doing a simple check - see `debugger.h`.